libs/sdk-backend-bear/src/backend/index.ts

```diff
@@ -343,7 +343,10 @@ export class BearBackend implements IAnalyticalBackend {
 
     private getAuthenticationContext = (): IAuthenticationContext => ({ client: this.sdk, backend: this });
 
-    private triggerAuthentication = (reset = false): Promise<IAuthenticatedPrincipal> => {
+    private triggerAuthentication = (
+        reset = false,
+        errorConverter: ErrorConverter = convertApiError,
```

Comment:
What is the use for this parameter? I don't see it used anywhere (passed a value).

Reply:
actually, looking through this i realized i made the fix unnecessary dangerous. will prepare rework shortly

Reply:
fix reworked